polaris-react/src/components/DataTable/DataTable.scss

```diff
@@ -337,3 +329,19 @@
     box-shadow: var(--p-shadow-base);
   }
 }
+
+.FixedFirstColumn {
+  position: absolute;
+
+  background: white;
+  @media #{$p-breakpoints-md-down} {
+    z-index: 1;
+  }
+
+  z-index: 3;
+  border-spacing: 0;
+
+  &.Visible th {
```

Comment:
Nit: `Visible` seems a bit misleading to me since it's technically always visible and it falsly implies that it is hidden otherwise.

What do you think about changing this to something like `&.showBorder`. It's a bit verbose, but expresses more clearly what this actually does (especially in the react code where I don't see the CSS properties).

Reply:
I'm not aware of a convention. I always make all `CSS` classes start with a capital letter.